### PR TITLE
Some style with some formatting 

### PR DESCRIPTION
**Pull request by itxsoumya**
Structured all README file content to tabular format + added a banner + added some emojis 

Screenshot from 2023-05-10 17-00-34

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,43 @@
...
+| :----: | :---: |
+| 01 | Create a ChatGPT clone using their APIs |
+| 02 | Use their APIs on the backend, relay the information to your frontend |
+| 03 | Make sure your frontend experience is 10x better than the native chat GPT experience. |
+| 04 | Allow users to put in their own Chat GPT API keys and use GPT 4. |
+
+<br><br>
+
+
+|  | 🔗 CREATE AN MULTI CHAIN AIRDROPPING WEBSITE 🔗 |
```
Comment (AdiSuyash) [inline README.md:38]: |     | 🔗 CREATE A MULTICHAIN AIRDROPPING WEBSITE 🔗 |

File: README.md [hunk 2]
```diff
@@ -1,25 +1,43 @@
...
+| 02 | Frontend should allow users to signup, create accounts and create a room |
+| 03 | On room creation, they can share the link with their friend who can join the group as well
+| 04 | Add move validation to make sure users can only make valid moves
+| 05 | Use either canvas or raw HTML blocks for the game. You may also use a game engine like phaser. |
+| 06 | You will need to do socket programming since this is a realtime game. |
+
+<br>
+<br> 
+
+|  | 🐤 PLAYER FLAPPY BIRD 🐤 |
```
Comment (AdiSuyash) [inline README.md:19]: |  | 🐤MULTIPLAYER FLAPPY BIRD GAME 🐤 |

**Review by AdiSuyash — CHANGES_REQUESTED** [review]
Please consider these changes.

File: README.md (hunk 1 above)
Comment (itxsoumya) [reply README.md]: thanks for noticing it.
mistakes have been resolved.

**Comment by hkirat** [comment]
Thankyou!

**Pull request merged**